prefs.js

```diff
@@ -169,6 +169,10 @@ var SettingsWidget = class SettingsWidget {
             this._settings.set_boolean('topbar-follow-focus',
                 state);
         });
+        if (this._settings.get_boolean('found-dash-to-panel')) {
+            topbarFollowFocus.set_sensitive(false);
+            topbarFollowFocus.get_parent().set_tooltip_text("Incompatible with Dash to Panel.");
+        }
```

Comment:
Ideally I'd like to follow HIG and add an icon next to the toggle to signify that something is up